File: packages/sentra-core/src/sentra/domain/entities/organization_entity.py
from sqlalchemy import String, Text
from sqlalchemy.orm import Mapped, mapped_column
from sentra.domain.entities.base_entity import BaseEntity


class OrganizationEntity(BaseEntity):
    __tablename__ = "organizations"

    name: Mapped[str] = mapped_column(String, nullable=False)
    slug: Mapped[str] = mapped_column(String, unique=True, nullable=False)
    description: Mapped[str] = mapped_column(Text, nullable=True)
    location: Mapped[str] = mapped_column(String, nullable=True)
    contact_email: Mapped[str] = mapped_column(String, nullable=True)

File: src/Sentra.Domain/Entities/OrganizationEntity.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace Sentra.Domain.Entities;

[Table("organizations")]
public class OrganizationEntity : BaseEntity
{
    public string Name { get; set; }
    public string Slug { get; set; }
    public string? Description { get; set; }
    public string? Location { get; set; }
    public string? ContactEmail { get; set; }
}
